src/components/search/entry/SortSelect.jsx

```diff
@@ -71,22 +71,31 @@ class SortSelect extends Component {
     } = this.props;
 
     return (
-      <select
-        className={styles.common}
-        value={value}
-        onChange={this.handleChange}
-      >
-        {
-          ['bestmatch', 'atoz', 'ztoa', 'newest', 'oldest'].map((sortOrder) => (
-            <option
-              key={sortOrder}
-              value={sortOrder}
-            >
-              {intl.formatMessage(messages[sortOrder])}
-            </option>
-          ))
-        }
-      </select>
+      <>
+        <label
+          className={styles.commonLabel}
```

Comment:
I don't think CSS is needed here. You should be able to put a space character between the label text and the `select`. If it is getting collapsed, you can use `{" "}` in JSX to ensure it gets rendered.

Reply:
Let know if what that I did is right please.